articles/multifactor-authentication/api/index.md

```diff
@@ -1,28 +1,25 @@
 ---
 title: Multifactor Authentication API
-description: Index of Available MFA APIs
+description: Overview of available multifactor authentication APIs
 ---
 
-# Multifactor Authentication (MFA) API
+# Multifactor Authentication API
 
-The Multifactor Authentication (MFA) API is a set of APIs that allows you to:
-
-* Enforce MFA when users interact with [the Token endpoints](/api/authentication#get-token)
-* Programmatically enroll and manage user authenticators
+The Multifactor Authentication (MFA) API endpoints allow you to enforce MFA when users interact with [the Token endpoints](/api/authentication#get-token), as well programmatically enroll and manage user authenticators.
 
 ## Requirements
 
-To use the MFA APIs you will need **an enabled MFA rule that sets Guardian as the MFA provider**. Fore more info on this see [Guardian for Administrators](/multifactor-authentication/administrator).
+To use the MFA APIs, an enabled MFA rule that sets Guardian as the MFA provider is required. For more information, see [Guardian for Administrators](/multifactor-authentication/administrator).
 
 If you are using the MFA API in conjunction with a [Token endpoint](/api/authentication#get-token), you must meet the requirements of the corresponding grant.
 
-## Enforcing MFA when interacting with the Token Endpoint
+## Multifactor authentication with the Token endpoint
 
 We have expanded MFA support on the Token endpoints to cover the following use cases:
 
-* Use MFA with the [password](/api-auth/grant/password), [password-realm](/api-auth/grant/password#realm-support), [refresh-token](/tokens/refresh-token/current#use-a-refresh-token) grants
-* Completion of first-time enrollment by end-users during authentication
-* Selection of the desired MFA authenticator by the end-user before they execute the MFA challenge
+* Use MFA with the [password](/api-auth/grant/password), [password-realm](/api-auth/grant/password#realm-support), and [refresh-token](/tokens/refresh-token/current#use-a-refresh-token) grants.
+* Completion of first-time enrollment by users during authentication.
+* Selection of the desired MFA authenticator by the user before they execute the MFA challenge.
 <!-- * Support for TOPT delivered via Email -->
```

Comment:
Are we tracking these pending TODOs in trello or similar? These are the next steps we need to execute.